ipatests/pytest_ipa/integration/__init__.py

```diff
@@ -494,15 +538,25 @@ def del_compat_attrs(cls):
     This is deprecated in favor of the mh fixture.
     To be removed when no more tests using this.
     """
-    del cls.master
-    del cls.replicas
-    del cls.clients
-    del cls.domain
-    if cls.ad_domains:
-        del cls.ads
-        del cls.ad_subdomains
-        del cls.ad_treedomains
-    del cls.ad_domains
+    if cls.num_trusted_domains == 1:
```

Comment:
Instead of having 2 separate code paths you can keep the old structure and add
```
    if cls.trusted_domains:
        del cls.trusted_master
        del cls.trusted_replicas
        del cls.trusted_clients
    del cls.trusted_domains
```